src/marketing/page-builder-add-product-recs.md

```diff
@@ -0,0 +1,101 @@
+---
+conditions: Default.EE-B2B
+title: Add Product Recommendations
+---
+
+Use the Product Recommendations content type to add an existing, active recommendation unit to the stage.
+
+## Add an existing recommendation unit
+
+1. Navigate to the Page Builder workspace on the target page.
+
+1. In the Page Builder panel under **Add Content**, drag a **Product Recommendation** placeholder to the stage.
+
+    ![]({% link images/images-ee/page-builder-add-prex-drag.png %}){: .zoom}
+    _Select Product Recommendation_
+
+1. Hover over the empty product recommendation container. Then, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}){: .Inline}) in the toolbox and do the following:
+
+    -  Click <span class="btn">Select</span>.
+
+        ![]({% link images/images-ee/page-builder-add-prex-select.png %}){: .zoom}
+        _Select Product Recommendation_
+
+    -  In the list of active product recommendations, find the row with the recommendation unit that you want to add. In the last column, click <span class="btn">Select</span>.
+
+        ![]({% link images/images-ee/page-builder-add-prex-selected.png %}){: .zoom}
+        _Selected Product Recommendation_
+
+        The name of the selected product recommendation appears in the **Edit Product Recommendation** form.
+
+        ![]({% link images/images-ee/page-builder-add-prex-name.png %}){: .zoom}
+        _Product Recommendation Name_
+
+    -  Complete the remaining settings as needed, using the field descriptions at the end of this page for reference.
+
+    -  When complete, click <span class="btn">Save</span> to return to the Page Builder workspace.
+
+1. In the upper-right corner, click **Save**.
+
+## Edit recommendation unit settings
+
+1. Hover over the recommendation unit container. Then, choose **Settings** (![]({% link images/images-ee/icon-pb-settings.png %}){: .Inline}) in the toolbox and do the following:
+
+    ![]({% link images/images-ee/page-builder-add-prex-toolbox.png %}){: .zoom}
+    _Block Toolbox_
+
+    -  To choose a different recommendation unit, click <span class="btn">Select Product Recommendation</span>. In the list of active recommendation units, <span class="btn">Select</span> the recommendation unit that you want to add. Then, click <span class="btn">Select</span>.
```

Comment:
We should not use buttons as verbs in an instruction. You have a `Select` button twice here and I am not sure how this actually works to make a suggestion.

Reply:
fixed